ui/imports/shared/popups/walletconnect/DAppConfirmDisconnectPopup.qml

```diff
@@ -0,0 +1,67 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import QtQml.Models 2.15
+
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+import StatusQ.Popups.Dialog 0.1
+
+StatusDialog {
+    id: root
+
+    objectName: "dappConfirmDisconnectPopup"
+
+    property string dappName
+    property url dappIcon
+    property string dappUrl
+
+    implicitWidth: 640
+
+    contentItem: StatusBaseText {
+        text: qsTr("Are you sure you want to disconnect %1 from all accounts").arg(dappUrl)
```

Comment:
Here we need to extract the domain and drop the `https://`. You can use `StringUtils.extractDomainFromLink`
In the subtitle as well.